components/rst-reader/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "habitat-rst-reader"
```

Comment:
I can see arguments on both sides, so I'm not saying this is "wrong", but I'm curious why you opted for a new crate, rather than adding another binary to the `butterfly` crate itself.

Reply:
It was literally the first thing that popped into my head.

Reply:
![tenor-1585137](https://user-images.githubusercontent.com/207178/58723355-dc481780-83a7-11e9-9511-2695e998cfae.gif)